packages/cb-check/cb-check.0.1/opam

```diff
@@ -0,0 +1,35 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Json schema checker for current-bench"
+maintainer: ["Ambre Austen Suhamy <ambre@tarides.com>"]
+authors: ["Ambre Austen Suhamy <ambre@tarides.com>"]
```

Comment:
The license field is missing, can you add it?

Reply:
Should be done now. Thanks for checking :)